crates/uv-resolver/src/requires_python.rs

```diff
@@ -32,7 +26,7 @@ pub struct RequiresPython {
     /// For a workspace, it's the union of all `requires-python` values in the workspace. If no
```

Comment:
That isn't the union, but the intersection, isn't it?

Reply:
Yes, good catch.